BridgeAppSDK/SBAUserWrapper+Bridge.swift

```diff
@@ -517,7 +517,14 @@ public extension SBAUserWrapper {
         }
     }
     
-    fileprivate func updateFromUserSession(_ response: SBAUserSessionInfoWrapper) {
+    /**
+     Call this any time the user session info gets updated.
+     */
+    public func updateFromUserSessionInfo(_ info: SBBUserSessionInfo) {
+        self.updateFromUserSession(info)
+    }
+    
+    private func updateFromUserSession(_ response: SBAUserSessionInfoWrapper) {
```

Comment:
not sure I understand the value of having both public and private instead of just making the wrapper one public.